src/main/flight/pid.c

```diff
@@ -205,21 +209,18 @@ void pidController(const pidProfile_t *pidProfile, const rollAndPitchTrims_t *an
     static float previousRateError[2];
     static float previousSetpoint[3];
 
-    if(FLIGHT_MODE(HORIZON_MODE))
-        calcHorizonLevelStrength(pidProfile);
```

Comment:
@martinbudden Well there is no point of running the same code twice, but I get the picture. You have done it to have rate more running as efficient as possible.
Perhaps you could now only run it on ROLL axis. That will make horizon mode also more efficient